### PR TITLE
Builder configs for cocoon try jobs.

### PR DESCRIPTION
**Pull request by godofredoc**
This is to enable presubmit tests for cocoon repo on luci.

https://github.com/flutter/flutter/issues/43348

### CODE REVIEW

File: config/main.star [hunk 1]
```diff
@@ -15,6 +15,7 @@ The documentation for lucicfg can be found here:
 https://chromium.googlesource.com/infra/luci/luci-go/+/refs/heads/master/lucicfg/doc/README.md
 """
 
+COCOON_GIT = 'https://chromium.googlesource.com/external/github.com/flutter/cocoon'
 FLUTTER_GIT = 'https://chromium.googlesource.com/external/github.com/flutter/flutter'
 ENGINE_GIT = 'https://chromium.googlesource.com/external/github.com/flutter/engine'
```
Comment (digiter) [inline config/main.star]: No need to address in this PR, do we want to move the repos out of chromium?

File: config/main.star [hunk 2]
```diff
@@ -298,6 +305,18 @@ linux_try_builder, linux_prod_builder = linux_builder()
...
 
+COMMON_COCOON_BUILDER_ARGS = {
+  'recipe': 'flutter/cocoon',
+  'console_view_name': 'cocoon',
+  'list_view_name': 'cocoon-try',
+}
+
+COCOON_EXTRAS = {
+  'caches': [swarming.cache(name='dart_pub_cache', path='.pub-cache')],
+}
```
Comment (digiter) [inline config/main.star]: Why this is in COCOON_EXTRAS, not COMMON_COCOON_BUILDER_ARGS?

File: config/main.star (hunk 1 above)
Comment (godofredoc) [reply config/main.star]: yes, but will need a little bit of experimentation and probably a bunch of new gerrit configs.

File: config/main.star (hunk 2 above)
Comment (godofredoc) [reply config/main.star]: Good point I was following the convention on the other configs but given that cocoon will only have linux configs it doesn't make sense to split the configs.

**Pull request merged**